src/vue/pages/Fees.spec.js

```diff
@@ -0,0 +1,96 @@
+import Fees from './Fees'
+
+import Vuex from 'vuex'
+
+import { vuexTypes } from '@/vuex'
+import walletModule from '@/vuex/wallet.module'
+
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+
+import { globalize } from '@/vue/filters/globalize'
+import { formatFeeType } from '@/vue/filters/formatFeeType'
+import { formatFeeSubType } from '@/vue/filters/formatFeeSubType'
+import { formatMoney } from '@/vue/filters/formatMoney'
+import { formatPercent } from '@/vue/filters/formatPercent'
+
+import { MockHelper, MockWrapper } from '@/test'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+localVue.filter('globalize', globalize)
+localVue.filter('formatFeeType', formatFeeType)
+localVue.filter('formatFeeSubType', formatFeeSubType)
+localVue.filter('formatMoney', formatMoney)
+localVue.filter('formatPercent', formatPercent)
+
+describe('Fees component unit test', () => {
+  let mockHelper
+  let wrapper
+  let feesResource
+  let feesSampleData = {
+    fees: {
+      ali: [0],
+      btc: [0, 1]
+    }
+  }
+
+  beforeEach(async () => {
+    sinon.restore()
+    mockHelper = new MockHelper()
+    const getters = walletModule.getters
+
+    sinon.stub(getters, vuexTypes.wallet).returns(
+      mockHelper.getMockWallet()
+    )
+
+    const store = new Vuex.Store({
+      modules: {
+        'new-wallet': {
+          namespaced: true,
+          getters
+        }
+      }
+    })
+
+    feesResource = mockHelper.getHorizonResourcePrototype('fees')
+    sinon.stub(feesResource, 'getAll').resolves(
+      MockWrapper.makeHorizonResponse(feesSampleData)
+    )
+
+    wrapper = await shallowMount(Fees, {
```

Comment:
stub the `created` hook directly on the Fees object inside the `beforeEach` hook. This will allow you to make tests more straightforward and you won't call the `loadFees` method when you don't need it.

for the tests of the created hook itself, use the `sinon.restore()` directly inside the test-case before you mount the component